package.json

```diff
@@ -59,7 +59,7 @@
     "eslint": "^7.23.0",
     "eslint-plugin-lit": "^1.3.0",
     "focus-visible": "^5.2.0",
-    "husky": "^4.3.8",
+    "husky": "^8.0.3",
```

Comment:
Be sure all these steps have been completed. Tested locally and pre-commit hook is not firing. 

https://github.com/AlaskaAirlines/auro-badge/issues/43